Pylint: C0325

```bash
mmpose/core/evaluation/mesh_eval.py:27:0: C0325: Unnecessary parens after 'assert' keyword (superfluous-parens)
mmpose/datasets/datasets/mesh/mesh_h36m_dataset.py:94:0: C0325: Unnecessary parens after 'assert' keyword (superfluous-parens)
```

--- a/mmpose/core/evaluation/mesh_eval.py
+++ b/mmpose/core/evaluation/mesh_eval.py
@@ -24,8 +24,8 @@ def compute_similarity_transform(source_points, target_points):
         source_points_hat (np.ndarray([N, 3])): Transformed source point set.
     """
 
-    assert (target_points.shape[0] == source_points.shape[0])
-    assert (target_points.shape[1] == 3 and source_points.shape[1] == 3)
+    assert target_points.shape[0] == source_points.shape[0]
+    assert target_points.shape[1] == 3 and source_points.shape[1] == 3
 
     source_points = source_points.T
     target_points = target_points.T

--- a/mmpose/datasets/datasets/mesh/mesh_h36m_dataset.py
+++ b/mmpose/datasets/datasets/mesh/mesh_h36m_dataset.py
@@ -91,7 +91,7 @@ class MeshH36MDataset(MeshBaseDataset):
         """Evaluate one example."""
         # Only 14 lsp joints are used for evaluation
         joint_mapper = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 18]
-        assert (joints_3d_visible[joint_mapper].min() > 0)
+        assert joints_3d_visible[joint_mapper].min() > 0
 
         pred_joints_3d = np.array(pred_joints_3d)
         pred_joints_3d = pred_joints_3d[joint_mapper, :]

